go-controller/pkg/ovn/default_network_controller.go

```diff
@@ -182,21 +157,21 @@ func newDefaultNetworkControllerCommon(cnci *CommonNetworkControllerInfo,
 		BaseNetworkController: BaseNetworkController{
 			CommonNetworkControllerInfo: *cnci,
 			controllerName:              DefaultNetworkControllerName,
-			NetConfInfo:                 &util.DefaultNetConfInfo{},
 			NetInfo:                     &util.DefaultNetInfo{},
+			NetConfInfo:                 &util.DefaultNetConfInfo{},
 			lsManager:                   lsm.NewLogicalSwitchManager(),
 			logicalPortCache:            newPortCache(defaultStopChan),
 			namespaces:                  make(map[string]*namespaceInfo),
 			namespacesMutex:             sync.Mutex{},
 			addressSetFactory:           addressSetFactory,
+			networkPolicies:             syncmap.NewSyncMap[*networkPolicy](),
+			sharedNetpolPortGroups:      syncmap.NewSyncMap[*defaultDenyPortGroups](),
+			podSelectorAddressSets:      syncmap.NewSyncMap[*PodSelectorAddressSet](),
 			stopChan:                    defaultStopChan,
 			wg:                          defaultWg,
 		},
-		externalGWCache:        make(map[ktypes.NamespacedName]*externalRouteInfo),
-		exGWCacheMutex:         sync.RWMutex{},
-		networkPolicies:        syncmap.NewSyncMap[*networkPolicy](),
-		sharedNetpolPortGroups: syncmap.NewSyncMap[*defaultDenyPortGroups](),
-		podSelectorAddressSets: syncmap.NewSyncMap[*PodSelectorAddressSet](),
+		externalGWCache: make(map[ktypes.NamespacedName]*externalRouteInfo),
+		exGWCacheMutex:  sync.RWMutex{},
```

Comment:
revert?

Reply:
didn't change it. it is only indentation change because we moved a few fields (sharedNetpolPortGroups etc.) into  BaseNetworkController.